doc/reference/network_bridge.md

```diff
@@ -121,13 +121,13 @@ The following features are supported for the `bridge` network type:
 - {ref}`network-forwards`
 - {ref}`network-zones`
 - {ref}`network-bgp`
-- {ref}`network-bridge-resolved`
+- [How to integrate with `systemd-resolved`](network-bridge-resolved)
 
 
 ```{toctree}
 :maxdepth: 1
 :hidden:
 
-Integrate with systemd-resolved </howto/network_bridge_resolved>
+Integrate with resolved </howto/network_bridge_resolved>
```

Comment:
shouldn't that also be `systemd-resolved` here?

Reply:
This is the issue with Sphinx removing `<code>` tags in links again. And in the text we talk about just "resolved" as well, and this looks better in the navigation anyway, so I think it's an improvement.